ui/main/package.json

```diff
@@ -53,6 +53,7 @@
     "moment-timezone": "0.5.44",
     "ng-event-source": "1.0.14",
     "ngx-chips": "3.0.0",
+    "ngx-quill": "24.0.4",
     "ol": "8.2.0",
     "quill": "1.3.7",
     "rrule": "2.8.1",
```

Comment:
instead of having quill import in two ways , i would prefer using the same solution as in build in template to manipulate 
quill component using :
` <opfab-richtext-editor id="myId"> </opfab-richtext-editor>`

The idea is to avoid having two solutions in the code base

Another advantage is that  we will be sure we share the same quill config (toolbarsOptions ...)